handlers/libhoney_event_handler.go

```diff
@@ -107,16 +108,21 @@ func (handler *libhoneyEventHandler) handleEvent(event assemblers.HttpEvent) {
 			Int64("request_id", event.RequestId).
 			Msg("Response timestamp is zero")
 		ev.AddField("http.response.timestamp_missing", true)
+		log.Info().Int64("response_request_id", event.RequestId)
```

Comment:
We already log request ID above.

```suggestion
```

Reply:
Whoops, left in unnecessary log statements. Updated in https://github.com/honeycombio/honeycomb-network-agent/pull/262/commits/0e154760fe9768a2d3a228ca1ab16c63b78ad263